### PR TITLE
Made rubber recipe more realistic

### PR DESCRIPTION
**Pull request by NCBPFluffyBear**
## Short Description
Modified rubber recipe to be more realistic (1 Oil Bucket now produces 6 rubber instead of 1)

## Additions/Changes/Removals
Changed the rubber synthesizer to produce 6 rubber instead of 1

## Related Issues
N/A

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with base Slimefun and made sure nothing breaks/unexpected happens.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added which may not be obvious to maintainers.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values

### CODE REVIEW

**Comment by J3fftw1** [comment]
Dont mark all of the checklist when you havent done everything

also, change it to 8 and imma merge it

**Comment by J3fftw1** [comment]
we also should output the bucket

File: src/main/java/dev/j3fftw/litexpansion/ItemSetup.java [hunk 1]
```diff
@@ -60,7 +60,18 @@ private void registerRubber() {
...
+            new RubberSynthesizer() {
+
+                @Override
+                public int getEnergyConsumption() {
+                    return 9;
+                }
+
+                @Override
+                public int getSpeed() {
+                    return 1;
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/litexpansion/ItemSetup.java]: why is this in Itemsetup

File: src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java [hunk 2]
```diff
@@ -2,148 +2,73 @@
...
     public static final RecipeType RECIPE_TYPE = new RecipeType(
-        new NamespacedKey(LiteXpansion.getInstance(), "scrap_machine"), Items.RUBBER_SYNTHESIZER_MACHINE
+        new NamespacedKey(LiteXpansion.getInstance(), "rubber_synthesizer"), Items.RUBBER_SYNTHESIZER
     );
-    private static final int INPUT_SLOT = 11;
-    private static final int OUTPUT_SLOT = 15;
-    private static final int PROGRESS_SLOT = 13;
-    private static final int PROGRESS_AMOUNT = 26; // Divide by 2 for seconds it takes
-    public static final int ENERGY_CONSUMPTION = 20_000 / PROGRESS_AMOUNT;
+    public static final int ENERGY_CONSUMPTION = 20_000 / 26;
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java:24]: what is 26

**Review by J3fftw1 — CHANGES_REQUESTED** [review]

File: src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java [hunk 3]
```diff
@@ -2,148 +2,73 @@
...
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.ArrayList;
+import java.util.List;
 
-public class RubberSynthesizer extends SlimefunItem implements InventoryBlock, EnergyNetComponent {
+import static io.github.thebusybiscuit.slimefun4.implementation.SlimefunItems.ADVANCED_CIRCUIT_BOARD;
+import static io.github.thebusybiscuit.slimefun4.implementation.SlimefunItems.REINFORCED_PLATE;
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java]: dont static import

File: src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java [hunk 4]
```diff
@@ -2,148 +2,73 @@
...
-        return new int[] {INPUT_SLOT};
+    public String getMachineIdentifier() {
+        return "RUBBER_SYNTHESIZER";
     }
 
     @Override
-    public int[] getOutputSlots() {
-        return new int[] {OUTPUT_SLOT};
+    public int getCapacity() {
+        return 256;
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java]: is this the same as before?

**Review by J3fftw1 — CHANGES_REQUESTED** [review]

File: src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java (hunk 2 above)
Comment (NCBPFluffyBear) [reply src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java:24]: It was the original ticks/item, was how it was calculated before so I didnt touch it

File: src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java (hunk 4 above)
Comment (NCBPFluffyBear) [reply src/main/java/dev/j3fftw/litexpansion/machine/RubberSynthesizer.java]: Defined it in a variable above but forgot to set

**Comment by J3fftw1** [comment]
Fix conflicts
Also does the machine have 2 output slots now?

**Comment by NCBPFluffyBear** [comment]
Yeah, all AContainers have 2 input 2 output which is why im not going to convert the scrap machine and mass fab (Also i dont think it can accept more than 1 input per output)

**Comment by NCBPFluffyBear** [comment]
Want a vid?

**Comment by NCBPFluffyBear** [comment]
https://streamable.com/3gaq8j

**Comment by J3fftw1** [comment]
Do old machines work?

**Comment by NCBPFluffyBear** [comment]
no xd

**Comment by NCBPFluffyBear** [comment]
might as well change it now while nobody has made it

**Comment by J3fftw1** [comment]
resolve conflicts

**Comment by J3fftw1** [comment]
conflicts

**Pull request merged**